src/main/java/org/gridsuite/shortcircuit/server/dto/ShortCircuitPredefinedConfiguration.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.shortcircuit.server.dto;
+
+/**
+ * @author AJELLAL Ali <ali.ajellal@rte-france.com>
```

Comment:
you ?

Reply:
same than https://github.com/gridsuite/shortcircuit-server/pull/81#discussion_r1629473725